.github/workflows/actions.yml

```diff
@@ -8,13 +8,13 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Checkout
-        uses: actions/checkout@v2
-      - name: Set up Cloud SDK
-        uses: google-github-actions/setup-gcloud@master
+        uses: actions/checkout@v3
+      - name: 'Authenticate Google Cloud with Service Account'
+        uses: google-github-actions/auth@v1
         with:
-          project_id: ${{ secrets.GCP_PROJECT_ID }}
```

Comment:
BQ users sometimes need `project_id` param.
e.g. there's service account in ETL/ELT GCP project, some tables & views in each microservices GCP project.

Reply:
updated

Reply:
Looks Good.